app/controllers/concerns/schools/placement_dates/wizard.rb

```diff
@@ -0,0 +1,46 @@
+module Schools
+  module PlacementDates
+    module Wizard
+      extend ActiveSupport::Concern
+
+      STEPS = %i[
+        placement_date
+        configuration
+        subject_selection
+      ].freeze
+
+    private
+
+      def next_step(placement_date, current_step = :placement_date)
+        next_step = find_next_step(current_step)
+        if next_step == :COMPLETED
+          placement_date.publish
+          auto_enable_school
+          redirect_to schools_placement_dates_path
+        else
+          redirect_to next_step_path(placement_date, next_step)
+        end
+      end
+
+      def find_next_step(current_step)
+        STEPS.drop(STEPS.index(current_step) + 1).detect(&method(:required?)) || :COMPLETED
```

Comment:
Probably a better way of doing `STEPS.drop(STEPS.index(current_step) + 1)`?

Reply:
```
STEPS[STEPS.index(current_step)..]
```